llvm/lib/Target/AIE/AIE2InstructionSelector.cpp

```diff
@@ -814,25 +818,23 @@ bool AIE2InstructionSelector::selectG_AIE_ADD_VECTOR_ELT_LEFT(
   unsigned Opcode;
   switch (VecEltDstTySize) {
   case 8:
-    Opcode = AIE2::VPUSH_LO_8;
+    Opcode = AIE2::VPUSH_HI_8;
```

Comment:
Now I think I am a bit confused.  Did we change G_AIE_ADD_VECTOR_ELT_LEFT to add to the right? Should we have GInst here?

Reply:
I would actually vouch for renaming this `G_AIE_ADD_VECTOR_ELT_HI`. "left" is too ambiguous

Reply:
[X] `G_AIE_ADD_VECTOR_ELT_HI`